playbook-website/.gitignore

```diff
@@ -23,10 +23,16 @@ public/assets
 # Ignore master key for decrypting credentials and more.
 config/master.key
 
-public/packs
-public/packs-test
 node_modules
 yarn-error.log
 yarn-debug.log*
 .yarn-integrity
-.npmrc
\ No newline at end of file
+.npmrc
+
+# Vite Ruby
+/public/vite*
+/public/cache*
```

Comment:
`tmp` was not writable at this time I added this besides the data added here is safe.